Library/Formula/nim.rb

```diff
@@ -1,16 +1,13 @@
-class Nimrod < Formula
+class Nim < Formula
   desc "Statically typed, imperative programming language"
   homepage "http://nim-lang.org/"
-  url "http://nim-lang.org/download/nim-0.11.2.tar.xz"
-  sha256 "5640e364d8bacec830f016daf3d4427911c48cebf962724ec903fea5d5a7a419"
+  url "http://nim-lang.org/download/nim-0.12.0.tar.xz"
+  sha256 "0fb860137f0888f6b1a593d9b115fd7057983b7355f4f9922bb82a49ac875235"
   head "https://github.com/Araq/Nim.git", :branch => "devel"
 
   bottle do
     cellar :any_skip_relocation
-    sha256 "3c9a33e6dd9e7caddd9a0daf8290a4cbf7eb5f483b9fdd5f42a78e8ca6f95396" => :el_capitan
```

Comment:
The CI handles bottle regeneration, not contributors. No need to tweak this block.